torch/_torch_docs.py

```diff
@@ -3332,6 +3332,8 @@ def merge_dicts(*dicts):
 of 2D square tensors, in which case this function would return a tensor composed of
 individual inverses.
 
+Supports real and complex input.
```

Comment:
Thanks for adding this "Supports..." statement. What about clarifying the complex behavior by updating it to:

"Supports real and complex input. Batches of complex matrices are only supported on CPU."

?

Reply:
Batches of complex matrices also work on CUDA, it's that they are not tested because batched matmul for complex doesn't work on CUDA. I'll update tests using for loop.